inst/examples/get_info.R

```diff
@@ -0,0 +1,6 @@
+file <- system.file("models", "sir.gaml", package = "rama")
+exp <- load_experiment("sir", file)
+get_info(exp, "Parameters", "name")
+get_info(exp, "Parameters", "type")
+get_info(exp, "Outputs", "name")
+get_info(exp, ".attrs", "experiment")
```

Comment:
I don't think it's necessary to have the function `get_info` adapted to the attributes as they can be easily accessed with: `attributes(exp)`. So, if everybody agrees, I think we can remove this line.